File: 프로그래머스/2/42578. 의상/의상.py
from collections import Counter
def solution(clothes):
    
    temp = []
    
    for i in clothes:
        temp.append(i[1])
        
    counter = Counter(temp)
    
    print(counter.values())
    
    answer = 1
    for i in counter.values():
        answer *= (i + 1)
    
    return answer - 1

File: 프로그래머스/2/42578. 의상/의상.java
import java.util.*;

class Solution {
    
    static int answer = 1;
    
    public int solution(String[][] clothes) {
        
        HashMap<String, Integer> map = new HashMap<>();
        
        for (String[] c : clothes) {
            map.put(c[1], map.getOrDefault(c[1], 0) + 1);
        }
        
        for (int count : map.values()) {
            answer *= (count + 1);
        }
        
        return answer - 1;
    }
}